CHANGELOG.md

```diff
@@ -2,11 +2,32 @@
 
 ## v2.0.0
 
+* Breaking change from 1.x: Removed deprecated methods
+  `is_in_european_union`, `is_anonymous`, `is_anonymous_vpn`,
+  `is_hosting_provider`, `is_public_proxy`, and `is_tor_exit_node`. The
+  non-deprecated equivalents are `in_european_union?`, `anonymous?`,
+  `anonymous_vpn?`, `hosting_provider?`, `public_proxy?`, and
+  `tor_exit_node?`.
+* Breaking change from 1.x: Removed deprecated methods for deprecated
+  attributes: `is_anonymous_proxy` and `is_satellite_provider`.
+* Breaking change from 1.x: Removed deprecated methods for deprecated
+  subscores: `email_tenure` and `ip_tenure`. For `email_tenure`, please use
+  the `email_address` subscore instead. For `ip_tenure`, please use
+  `risk_score` instead.
+* Breaking change from 1.x: Removed deprecated method for deprecated
+  attribute `ip_address.country.is_high_risk`.
```

Comment:
Is this replaced with `high_risk?`? If so, maybe it would be worth mentioning that.

Reply:
It's not available at all now with this change. Would it be similar to `is_anonymous_proxy`/`is_satellite_provider` that you think we should still expose it despite being deprecated?